explorer/app/components/Index/common/constants.ts

```diff
@@ -71,11 +72,129 @@ export const BIND_SUMMARY_RESPONSE = {
     getSummaryCount(r, SUMMARY_KEY.SPECIMENS),
 };
 
+export const NETWORK_KEYS = [
+  "Adipose",
+  "Breast",
+  "Development",
+  "Eye",
+  "Genetic-diversity",
+  "Gut",
+  "Heart",
+  "Immune",
+  "Kidney",
+  "Liver",
+  "Lung",
+  "Musculoskeletal",
+  "Nervous-system",
+  "Oral",
+  "Organoid",
+  "Pancreas",
+  "Reproduction",
+  "Skin",
+] as const;
+
+export const NETWORKS: Network[] = [
```

Comment:
We probably do not need `NETWORKS`, given the returned value from API is what we display e.g. "Adipose".